README.md

```diff
@@ -24,6 +24,18 @@ custom:
     distributionFolder: client/dist # (Optional) The location of your website. This defaults to client/dist
     indexDocument: index.html # (Optional) The name of your index document inside your distributionFolder. Defaults to index.html
     errorDocument: error.html # (Optional) The name of your error document inside your distributionFolder. Defaults to error.html
+    objectMetadata: # (Optional) S3 object metadata (HTTP headers) for hosted files
+      ALL_OBJECTS: # (Optional) List of headers to set for all files in bucket
+        - name: Content-Language # HTTP header name
+          value: en-US # header value
+      specific-directory/: # List of headers to set for files in specific-directory (path relative to distributionFolder). Overrides ALL_OBJECTS headers
```

Comment:
I think we should also be explicit that the trailing slash is how an options is determined to be a folder or not, to avoid this sort of situation:
```
objectMetadata:
    public: # I'm expecting this to be my 'public' folder, not a file called 'public'
      - headerName: Cache-Control
        value: max-age=10000
```

Reply:
Good point--I'll change this